src/config.js

```diff
@@ -0,0 +1,6 @@
+const authenticator = require('./services/authenticator');
+
+module.exports = {
+  authToken: authenticator.getAuthToken(),
+  serverHost: process.env.SERVER_HOST ? process.env.SERVER_HOST : 'https://api.forestadmin.com',
+}
```

Comment:
Moving this nice "refactoring" component in a dedicated PR would reduce the diff of this PR and make it lighter.

Reply:
So, do you want me to move it?